backend/vaa-strapi/src/api/answer/routes/answer.ts

```diff
@@ -3,53 +3,44 @@
  */
 
 import { factories } from '@strapi/strapi';
-import { restrictResourceOwnedByCandidate, restrictPopulate, restrictFilters, electionCanEditAnswers } from '../../../util/acl';
+import {
+  electionCanEditAnswers,
+  restrictFilters,
+  restrictPopulate,
+  restrictResourceOwnedByCandidate
+} from '../../../util/acl';
+import { Generic } from '../../../util/acl.type';
```

Comment:
Incomplete typing for the ACL functions is also fixed. This `Generic` type is something of an ugly workaround Strapi's incomplete typing. See more in the comment for the `util/acl` file.